runners/pc-usbip/Makefile

```diff
@@ -0,0 +1,45 @@
+APPNAME=usbip-simulation
+APPPATH=target/debug/usbip-simulation
+
+all: | start-sim attach help
+
+.PHONY:
+help:
+	@echo "###################################################"
+	@echo "Done. Device should be visible in your system now. Run 'make stop' to disconnect it."
+
+.PHONY: start-sim
+start-sim: $(APPNAME)
+	-$(MAKE) stop
+	cp $(APPPATH) ./$(APPNAME) -v
+	env RUST_LOG=debug ./$(APPNAME) &
```

Comment:
Did you consider using `cargo run` instead of copying around the binary and executing it directly?  If you don’t want to use `cargo run`, taking `CARGO_TARGET_DIR` into account when setting `APPPATH` (or making it possible to override `APPPATH`) would help users using a non-default target directory (for example me :)).